### ISSUE TITLE
Enable pyflakes on Travis-CI

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8486]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8486|
|Type|enhancement|
|Created|2016-06-19 14:41:42Z|
|Branch|https://github.com/twisted/twisted/tree/8486-travis-pyflakes|

pyflakes can now be run on a local branch and will not produce any false positives.

we should enable it in Travis-CI

We should also convert it to a simple builder in buildbot

<details><summary>Searchable metadata</summary>

```
trac-id__8486 8486
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__8486_travis_pyflakes 8486-travis-pyflakes
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1466347302944901 1466347302944901
changetime__1466549767191816 1466549767191816
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban removed owner|
|-|-|

I have added pyflakes to travis. I have also updated pyflakes to also check the bin and admin folders...

pyflakes will only check for *.py files in bin and admin... but is a start.

See https://github.com/twisted/twisted/pull/199 for Travis results.

Sending this to buildbot

Please check the changes. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @adiroiban|
|-|-|

Looks OK.  Ready to land.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 51f8c99860b569278a9b8bbcbc44a5c63be5d620
```
#!CommitTicketReference repository="" revision="51f8c99860b569278a9b8bbcbc44a5c63be5d620"
Merge 8486-travis-pyflakes: Enable pyflakes on Travis-CI.

Author: adiroiban
Reviewer: rodrigc
Fixes: #8486
```
